src/gui/CMakeLists.txt

```diff
@@ -228,6 +236,7 @@ ENDIF (NOT WIN32)
 
 INCLUDE_DIRECTORIES(
 ${CMAKE_CURRENT_SOURCE_DIR}
+${CMAKE_CURRENT_SOURCE_DIR}/anari
```

Comment:
I took a look at anari's anari_Exports.cmake. It specifies INTERFACE_INCLUDE_DIRECTORIES for its targets, so there should be no need to add anari to INCLUDE_DIRECTORIES.

Reply:
It's needed as VisIt will fail to compile if it's not there. However, it shouldn't be added directly to `include_directories` without first checking `HAVE_ANARI`. I've moved it to the `if(HAVE_ANARI)` block.